MAINTAINERS.md

```diff
@@ -17,5 +17,6 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Tejas Shah              | [shatejas](https://github.com/shatejas)               | Amazon      |
 | Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)                 | Amazon      |
 | Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)               | Amazon      |
+| Vikasht34               | [Vikasht34](https://github.com/Vikasht34)             | Amazon      |
```

Comment:
nit: Can we add his full name like others instead of github id ?

Reply:
oh my bad.

Reply:
![Uploading Screenshot 2025-02-03 at 11.14.16 AM.png…]()